### PR TITLE
Forward calculation

### PR DESCRIPTION
**Pull request by licode**
Make sure the forward calculation of spectrum is correct. 
To be merged soon.

### CODE REVIEW

File: pyxrf/view/fit.enaml [hunk 1]
```diff
@@ -382,11 +385,13 @@ enamldef ElementEdit(Window):
                             cb_strat3, cb_strat4, cb_strat5, cb_strat6, spacer)
                         ]
                         FloatField: ff_value:
-                            value := float(parameter_model.param_dict[loop_item]['value'])
```
Comment (ericdill) [inline pyxrf/view/fit.enaml]: did you figure out how to fix this float problem?

Comment (licode) [inline pyxrf/view/fit.enaml]: Yes. Some values are mistakenly created as int, so I corrected that part. Everything is working now. We do not need to put float here. 
I also find if we put float here, the dict value will not change if value changes from the GUI.

Comment (ericdill) [inline pyxrf/view/fit.enaml]: Ah, great

**Comment by ericdill** [comment]
Li, I've managed to nuke the majority of the unrelated commits. I've cherry picked these commits locally and will force push to master.

**Pull request closed without merging**